doc/rbd/index.rst

```diff
@@ -4,18 +4,19 @@
 
 .. index:: Ceph Block Device; introduction
 
-A block is a sequence of bytes (often 512).
+RBD (the Ceph block device) is a service that provides thin-provisioned,
+resizable block devices whose features include snapshotting and cloning. 
+
 Block-based storage interfaces are a mature and common way to store data on
-media including HDDs, SSDs, CDs, floppy disks, and even tape.
-The ubiquity of block device interfaces is a perfect fit for interacting
-with mass data storage including Ceph.
+media including HDDs, SSDs, CDs, floppy disks, and even tape. The ubiquity of
+block device interfaces is a perfect fit for interacting with mass data storage
+including Ceph.
 
-Ceph block devices are thin-provisioned, resizable, and store data striped over
-multiple OSDs.  Ceph block devices leverage
+RBD stores data striped over multiple OSDs. Ceph block devices leverage
 :abbr:`RADOS (Reliable Autonomic Distributed Object Store)` capabilities
-including snapshotting, replication and strong consistency. Ceph block
-storage clients communicate with Ceph clusters through kernel modules or
-the ``librbd`` library.
+including snapshotting and replication, and they provide strong consistency.
+Ceph block storage clients communicate with Ceph clusters by means of kernel
+modules or the ``librbd`` library.
```

Comment:
Per previous suggestion: "by means of a Linux kernel module"

Reply:
Roger that, @idryomov. I've included this elaboration.